share/setup.nsi.in

```diff
@@ -20,7 +20,7 @@ SetCompressor /SOLID lzma
 !define MUI_STARTMENUPAGE_REGISTRY_KEY ${REGKEY}
 !define MUI_STARTMENUPAGE_REGISTRY_VALUENAME StartMenuGroup
 !define MUI_STARTMENUPAGE_DEFAULTFOLDER "@PACKAGE_NAME@"
-!define MUI_FINISHPAGE_RUN $INSTDIR\bitcoin-qt.exe
+!define MUI_FINISHPAGE_RUN $INSTDIR\@PACKAGE_TARNAME@-qt.exe
```

Comment:
Hmm, if we're parametrizing executable names anyway I'd prefer to be able to customize the entire name, e.g. have a `@BITCOIN_QT_NAME@`, and set that to `@PACKAGE_TARNAME@-qt` somewhere higher in the build system.

There's been talk of getting rid of the `-qt` suffix for the user interface in the past, so having it hard-coded in as few places as possible would be a step forward.